src/constants.ts

```diff
@@ -10,6 +10,7 @@
 
 export const DEFAULT_CHE_IMAGE = 'quay.io/eclipse/che-server:nightly'
 export const DEFAULT_CHE_OPERATOR_IMAGE = 'quay.io/eclipse/che-operator:nightly'
+export const DEFAULT_DEV_WORKSPACE_CONTROLLER_IMAGE = 'quay.io/devfile/devworkspace-controller:next'
```

Comment:
Do we need to update it on release https://github.com/che-incubator/chectl/blob/master/make-release.sh#L60 ?

Reply:
done